ios/make_ios_project.sh

```diff
@@ -95,7 +95,7 @@ if [ -f "${infoplist}" ]; then
 	#	echo "Encountered error adding custom key UIBackgroundModes to plist!"
 	#	exit 1
 	#fi
-	longver=`git describe --tags`
+	longver=`git_describe_filtered 'ios_*'`
```

Comment:
This script doesn't see `contrib/base.sh` -- it pre-dates it and i'd rather not mix them together now because who knows what dragons lay there. 

Can you just remove this from this commit?  

I'll survive on the ios version without this feature... since anyway the ios version requires you to manually do all this crap to get the right stuff to work with apple -- i always manually set the app version in info.plist anyway ...

Reply:
Oh, right, I didn't notice that. I'll take it out of the patch.

Reply:
Updated